app/models/image.rb

```diff
@@ -19,6 +22,19 @@ class Image < ActiveRecord::Base
   scoped_search :relation => :operatingsystem, :on => :id, :rename => "operatingsystem", :complete_enabled => false, :only_explicit => true, :validator => ScopedSearch::Validators::INTEGER
   scoped_search :on => :user_data, :complete_value => {:true => true, :false => false}
 
+  def password_changed_changed?
```

Comment:
Proper solution is to add a support in audited gem to handle password fields (no monkey patching). This is a security fix so I'd like to avoid touching other code and do the refactoring in here, we might need to bring it back to older versions of Foreman. Btw there is already an upstream issue for this - https://github.com/collectiveidea/audited/issues/54 feel free to upvote.